MusicCloud/MusicCloud/Scenes/ViewControllers/Splash/SplashViewController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SplashViewController.swift
+//  MusicCloud
+//
+//  Created by thanh on 2/21/19.
+//  Copyright © 2019 thanh. All rights reserved.
+//
+
+import UIKit
+
+class SplashViewController: UIViewController {
```

Comment:
```suggestion
final class SplashViewController: UIViewController {
```
vì class này không có subclass nên để key `final` để ngăn chặn override